apps/android/app/src/main/java/chat/simplex/app/views/chat/ChatInfoView.kt

```diff
@@ -110,13 +112,13 @@ fun ServerImage(chat: Chat) {
   val status = chat.serverInfo.networkStatus
   when {
     status is Chat.NetworkStatus.Connected ->
-      Icon(Icons.Filled.Circle, "Connected", tint = MaterialTheme.colors.primaryVariant)
+      Icon(Icons.Filled.Circle, generalGetString(R.string.server_connection_status_connected), tint = MaterialTheme.colors.primaryVariant)
```

Comment:
this is content description - should we translate it too? probably so, as it is for screen readers?